client/client.go

```diff
@@ -655,6 +655,28 @@ func (client *Client) CreateUsers(options []*CreateUserOptions) ([]*CreateUserRe
 	return results, nil
 }
 
+// RemoveUserOptions holds options for removing a local system user.
+type RemoveUserOptions struct {
+	Username string `json:"username,omitempty"`
+}
+
+// RemoveUser removes a local system user.
+func (client *Client) RemoveUser(options *RemoveUserOptions) error {
+	if options.Username == "" {
```

Comment:
Should we check if `options` are nil?

Reply:
If we do, we should also check in CreateUser(). RemoveUser() is only called from the user command handler and there it's guaranteed to be non-null. I can add the test if you feel it's necessary, but in the current scenario it seems that we're safe.

Reply:
a `nil` would be a programming error, and I can think of no clearer way of telling the developer than `panic`ing :-)

Reply:
While I would otherwise agree we have some cases where options-like argument can be nil to mean an all-zero struct internally. Perhaps username should not be in the options if it is not ... you know.. optional.